<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INBNRMARITA" failures="3" tests="17" name="Farmer Onboarding" time="202.619" errors="0" timestamp="2022-02-15T18:09:34 IST">
  <testcase classname="script.CreatePO" name="buyerLinkValidation" time="3.751"/>
  <testcase classname="script.CreatePO" name="buyerOnboardingPage" time="4.665"/>
  <testcase classname="script.CreatePO" name="searchButton" time="12.267"/>
  <testcase classname="script.CreatePO" name="searchUser" time="10.092"/>
  <testcase classname="script.CreatePO" name="userCardValidation" time="10.077"/>
  <testcase classname="script.CreatePO" name="addCropButton" time="10.081"/>
  <testcase classname="script.CreatePO" name="validateCategory" time="10.08"/>
  <testcase classname="script.CreatePO" name="validateQuantity" time="10.046"/>
  <testcase classname="script.CreatePO" name="validateUnit" time="10.033"/>
  <testcase classname="script.CreatePO" name="validatePrice" time="10.06"/>
  <testcase classname="script.CreatePO" name="addCrop" time="10.05"/>
  <testcase classname="script.CreatePO" name="validateAddCropButton" time="10.048"/>
  <testcase classname="script.CreatePO" name="poDates" time="11.051"/>
  <testcase classname="script.CreatePO" name="deliveryDistrictValidation" time="21.079"/>
  <testcase classname="script.CreatePO" name="cancelPoValidation" time="10.04">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at script.CreatePO.cancelPoValidation(CreatePO.java:435)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- cancelPoValidation -->
  <testcase classname="script.CreatePO" name="generatePO" time="20.547">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at script.CreatePO.generatePO(CreatePO.java:455)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- generatePO -->
  <testcase classname="script.CreatePO" name="poCreation" time="21.615">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at script.CreatePO.poCreation(CreatePO.java:474)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- poCreation -->
</testsuite> <!-- Farmer Onboarding -->
